src/visualizr/anitalker/networks/encoder.py

```
@@ -240,25 +240,21 @@ def __init__(self, in_channel, out_channel):
 
 class WeightedSumLayer(nn.Module):
     def __init__(self, num_tensors=8):
         super(WeightedSumLayer, self).__init__()
-
         self.weights = nn.Parameter(randn(num_tensors))
 
     def forward(self, tensor_list):
         weights = softmax(self.weights, dim=0)
-        weighted_sum = zeros_like(tensor_list[0])
+        weighted_sum: Tensor = zeros_like(tensor_list[0])
         for _tensor, weight in zip(tensor_list, weights):
```

Ruff (0.12.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)